validator/test/fixtures/constraints/iam_allowed_policy_member_reject_project_reference/data.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: constraints.gatekeeper.sh/v1alpha1
+kind: GCPIAMAllowedPolicyMemberDomainsConstraint
+metadata:
+  name: reject_project_reference
+spec:
+  severity: high
+  match:
+    target: ["organization/*"]
+    exclude: [] # optional, default is no exclusions
+  parameters:
+    allow_project_references: false
```

Comment:
is project reference the right name for those iam bindings? doesn't sound intuitive from a user point of view.

Reply:
good point. I looked at https://cloud.google.com/storage/docs/access-control/iam#acls, and it just refers to them as some specific member types that are different from the typical ones. 
But that got me thinking that instead of using boolean parameter, we could use a whitelist of IAM member types. That's more explicit and flexible. I've made this change; PTAL.